### ISSUE TITLE
the model does not provide any valid output, check model architecture and the data input

### ISSUE THREAD

**Issue by SakuraMaxuetao** [zylo117/Yet-Another-EfficientDet-Pytorch#468]

**Comment by SakuraMaxuetao** [comment]
Thanks for you sharing , I hava a problem but i can`t solve it.
After 1 epoch ,all loss become nan ,could you give me some advise?
Epoch: 1/500. Classification loss: nan. Regression loss: nan. Total loss: nan
WARNING:root:NaN or Inf found in input tensor.

**Comment by SakuraMaxuetao** [comment]
Is there something wrong with my dataset？ In my dataset, every supercategory have several different  subcategory（name）, just like this:
"categories": [
        {
            "supercategory": "BB",
            "id": 0,
            "name": "BB1"
        },
        {
            "supercategory": "BB",
            "id": 1,
            "name": "BB2"
        },
In the project_name.yml and efficientdet/config.py,  we need to change  the obj_list and COCO_CLASSES, should I use the name of the supercategory or subcategory？

**Comment by zylo117** [comment]
All you need to modify is `project.yml`. `supercategory` is not supported here.

**Comment by SakuraMaxuetao** [comment]
I've solved this problem. Sorry, that's a low-level error: I have too few validation sets.

**Comment by SakuraMaxuetao** [comment]
![微信图片_20200811102515](https://user-images.githubusercontent.com/24203678/89850479-05278780-dbbd-11ea-95ec-79ba9b2cb994.png)
Sorry to ask again，i met this question:Exception: the model does not provide any valid output, check model architecture and the data input. It seems that someone else has encountered this problem, but he has not solved it

**Comment by zylo117** [comment]
@SakuraMaxuetao probably it's underfitting, keep training till loss won't decrease

**Comment by SakuraMaxuetao** [comment]
I trained 500 epochs, but the problem is still unsolved. My dataset comes from ：https://public.roboflow.ai/object-detection/chess-full

**Comment by kimalaacer** [comment]
> I trained 500 epochs, but the problem is still unsolved. My dataset comes from ：https://public.roboflow.ai/object-detection/chess-full

I am having the same problem. 
how did you fix this error: 
Exception: the model does not provide any valid output, check model architecture and the data input

**Comment by zylo117** [comment]
How about this. If the dataset is not too large and you're willing to open the dataset or it's open already, you can post the dataset in the format of this repo required. I might try to train it for you.
